modules/virtualization/microvm/guivm.nix

```diff
@@ -21,7 +21,7 @@
           users.accounts.enable = lib.mkDefault configHost.ghaf.users.accounts.enable;
           profiles.graphics.enable = true;
           # Uncomment this line to take LabWC in use
-          # profiles.graphics.compositor = "labwc";
+          profiles.graphics.compositor = "labwc";
```

Comment:
We can't yet this to default to "labwc" so let's keep it just configurable to "labwc" until further development. We use the same - not default to new features - with other security features under development as well.

Reply:
I agree - we need to make sure everything (and everyone) is ready to switch to `labwc` first.